awx_collection/plugins/module_utils/controller_api.py

```diff
@@ -446,14 +423,21 @@ def get_one(self, endpoint, name_or_id=None, allow_none=True, **kwargs):
             # Or we weren't running with a or search and just got back too many to begin with.
             self.fail_wanted_one(response, endpoint, new_kwargs.get('data'))
 
+        if check_exists:
+            name_field = self.get_name_field_from_endpoint(endpoint)
+            item_id = response['json']['results'][0]['id']
+            self.json_output['id'] = item_id
```

Comment:
In addition to ID, can we get name in the output, right now ALL it outputs is:
```
{
"result": {
    "changed": false,
    "failed": false,
    "id": 1
}
}
```
Should be something like 
```
If response['json']['results'][0]['name']
    self.json_output['name'] = response['json']['results'][0]['name']
```

Also Shouldn't this be one line?
```
self.json_output['id'] = response['json']['results'][0]['id']
````

Reply:
I ran a little test playbook on an org and we don't return name during cretion/no change/edit or delete:

```
$ ansible-playbook test_organization.yml 
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [Test org creation] *******************************************************************************************************************************************************************************************************************************

TASK [Create a new org] ********************************************************************************************************************************************************************************************************************************
[WARNING]: You are running collection version 0.0.1-devel but connecting to AWX version 22.1.1.dev33+gc10ada6f44
ok: [localhost]

TASK [debug] *******************************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": {
        "changed": false,
        "failed": false,
        "id": 26,
        "warnings": [
            "You are running collection version 0.0.1-devel but connecting to AWX version 22.1.1.dev33+gc10ada6f44"
        ]
    }
}

TASK [Create an existing org] **************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [debug] *******************************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": {
        "changed": false,
        "failed": false,
        "id": 26,
        "warnings": [
            "You are running collection version 0.0.1-devel but connecting to AWX version 22.1.1.dev33+gc10ada6f44"
        ]
    }
}

TASK [Exist an org] ************************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [debug] *******************************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": {
        "changed": true,
        "failed": false,
        "id": 26,
        "warnings": [
            "You are running collection version 0.0.1-devel but connecting to AWX version 22.1.1.dev33+gc10ada6f44"
        ]
    }
}

TASK [Delete an org] ***********************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [debug] *******************************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": {
        "changed": false,
        "failed": false,
        "id": 26,
        "warnings": [
            "You are running collection version 0.0.1-devel but connecting to AWX version 22.1.1.dev33+gc10ada6f44"
        ]
    }
}

PLAY RECAP *********************************************************************************************************************************************************************************************************************************************
localhost                  : ok=8    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

```

So I'd like to stay consistent. 

I'll make that other call one line, no reason for it not to be.